src/shared/components/dateRangePicker/DateRangePicker.tsx

```diff
@@ -108,11 +123,27 @@ class DateRangePicker extends PureComponent<Props, State> {
   }
 
   private handleSelectLower = (lower: string): void => {
-    this.setState({lower})
+    this.setState(prev => ({
+      ...prev,
```

Comment:
why are we concerned with setting the previous stuff?

Reply:
My bad, I had some bugs that I thought were because I was doing setState wrong, turned out the issue was something else. I just forgot to delete it.